mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -271,6 +260,20 @@ def _rpc(self, method, *args):
             raise ProtocolError(ProtocolError.MISMATCHED_API_ID)
         return result['result']
 
+    def _is_app_running(self):
```

Comment:
It's hard to change the RPC protocol because of sl4a. If we add a new message the framework relies on, we'd have to port it to sl4a and then backport it to all supported branches. :(